runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -465,6 +465,7 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          vmInfo._interpreterVTableOffset = TR::Compiler->vm.getInterpreterVTableOffset();
          vmInfo._maxHeapSizeInBytes = TR::Compiler->vm.maxHeapSizeInBytes();
          vmInfo._enableGlobalLockReservation = vmThread->javaVM->enableGlobalLockReservation;
+         vmInfo._prepareForOSRHelper = comp->getSymRefTab()->findOrCreateRuntimeHelper(TR_prepareForOSR, false, false, false)->getMethodAddress();
```

Comment:
In other parts of the code I see the following parameters being passed:
`findOrCreateRuntimeHelper(TR_prepareForOSR, false, false, true);`
This is the meaning of the parameters:
```
TR::SymbolReference *
OMR::SymbolReferenceTable::findOrCreateRuntimeHelper(TR_RuntimeHelper index,
                                                     bool             canGCandReturn,
                                                     bool             canGCandExcept,
                                                     bool             preservesAllRegisters)
```
Is there a reason you used `false` for `preserveAllregisters`?

Reply:
I just saw that all other helpers saved in VM info use this parameters so I just copied that. I'll change it to the parameters used elsewhere for this helper.

Reply:
This code is obsolete in the new implementation